gateway/cli/src/main.rs

```diff
@@ -23,12 +25,13 @@ pub enum Commands {
     Info,
     /// Check gateway balance
     /// TODO: add federation id to scope the federation for which we want a pegin address
-    Balance,
+    Balance { federation_id: String },
```

Comment:
Can we use the newtype instead here? I think it either only needs an annotation or an impl of `FromStr` to work.